src/common.jl

```diff
@@ -75,7 +75,11 @@ function _get_withdrawals!(
     bus_lookup::Dict{Int, Int},
     sys::PSY.System,
 )
-    loads = PSY.get_components(x -> !isa(x, PSY.FixedAdmittance), PSY.ElectricLoad, sys)
+    loads = PSY.get_components(
+        x -> (!isa(x, PSY.FixedAdmittance) && !isa(x, PSY.SwitchedAdmittance)),
```

Comment:
This would require changes to PSY and might not be your problem but maybe it would be good to put `FixedAdmittance` and `SwitchedAdmittance` under a common supertype if they're going to be filtered on together like this?

Reply:
@GabrielKS lets open a PR to review the type hierarchy in PSY and update before we merge this code